README.md

```diff
@@ -1 +1,34 @@
-# Gruvo
\ No newline at end of file
+# Gruvo
+This is an educational project - a twitter replica. 
+
+## Getting Started
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. 
+
+### Prerequisites
+1. .Net core sdk
+2. Either VSCode with C# extension OR Visual studio 2017
+3. Nodejs
+
+### Installing
+1. Clone the repo
+    git clone https://github.com/AleksMorozova/Gruvo.git
+2. Create database 
+    Execute script from "script for creation of DB" folder on sql server
+3. Add connection string to appsetings.json
+    For instance, 
+```{ 
+    "ConnectionStrings": {
```

Comment:
Didn't work for me. Connection string should also be set up in startup, I think. Will be refactored under #23 .